exonum-light-client/README.md

```diff
@@ -78,15 +86,24 @@ HashCode txHash = exonumClient.submitTransaction(tx);
 ```
 *Be aware that this method submits the transaction to the pool of
 uncommitted transactions and dosn't wait for the transaction 
-acceptance to a new block.* 
+acceptance to a new block.*  
  
+Also, you can take a look at the [integration test][send-tx-it]
+for the full example of how to create a transaction message and
+send it to Exonum node.
 
 ## How to build
 To build the client locally, clone the repository, and
-run next command from the project's root 
+run next commands from the project's root 
 i.e. _exonum-java-binding_ directory:
 ```bash
-source exonum-java-binding/tests_profile
+cd exonum-light-client
+mvn install
+```
+In case you need an unreleased version of the [`common`][common-mvn] module
```

Comment:
I'd leave no options — set RUSTFLAGS, run this command (it will always work). How do I know if I need an unreleased version of the common module?

We can add a todo to address that later.